ogc.py

```diff
@@ -0,0 +1,103 @@
+import time

```

Comment:
remove file from root dir

Reply:
Thanks. I've removed the file from root dir.